spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Friendship, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
If you're not going to add tests, delete this.... but of course you are going to add tests.